adapters/taboola/taboola.go

```diff
@@ -131,7 +131,14 @@ func (a *adapter) buildRequest(request *openrtb2.BidRequest) (*adapters.RequestD
 		return nil, fmt.Errorf("unsupported media type for imp: %v", request.Imp[0])
 	}
 
-	url, err := a.buildEndpointURL(request.Site.ID, mediaType)
+	var taboolaPublisherId string
+	if request.Site != nil && request.Site.ID != "" {
+		taboolaPublisherId = request.Site.ID
+	} else if request.App != nil && request.App.ID != "" {
+		taboolaPublisherId = request.App.ID
+	}
+
+	url, err := a.buildEndpointURL(taboolaPublisherId, mediaType)
```

Comment:
Please add a json test case for this change.

Reply:
@gargcreation1992  I did added a test file 
`bannerAppRequest.json`